### ISSUE TITLE
Error starting Vagrant box on v0.3.0

### ISSUE THREAD

**Issue by avade** [cloudfoundry-incubator/lattice#176]
```
==> default: stdin: is not a tty
==> default: Running provisioner: shell...
    default: Running: /var/folders/nr/r3_6p6n14819b_k1s3xtwfgm0000gn/T/vagrant-shell20150816-1561-ej580g
==> default: stdin: is not a tty
==> default: Downloading https://s3-us-west-2.amazonaws.com/lattice/releases/v0.3.0/lattice.tgz ..
==> default: DONE!
==> default:  * Stopping web server lighttpd
==> default:    ...done.
==> default:  * Starting web server lighttpd
==> default:    ...done.
==> default:
==> default: gzip: stdin: unexpected end of file
==> default: tar: Unexpected EOF in archive
==> default: tar: Unexpected EOF in archive
==> default: tar:
```

When checking out the  `v0.2.7 ` tag and running `vagrant up` Lattice comes up fine.

```
Vagrant 1.7.4
Virtualbox 5.0.2
Mac OSX 10.9.3
```

**Comment by menicosia** [comment]
Hi @avade,

My first instinct is that something went wrong with the automated download of the lattice.tgz file:
> ==> default: Downloading https://s3-us-west-2.amazonaws.com/lattice/releases/v0.3.0/lattice.tgz ..

We have been using v0.3.0 for some time, so we know that this error is not typical:
> ==> default: gzip: stdin: unexpected end of file

Do you know if this happens repeatedly? I'm hoping this is a network or HTTP proxy glitch, rather than an endemic problem. (Pretty sure that even if you're in the UK, you should be able to d/l from S3 just fine!)

--
  Marco Nicosia
  Product Manager
  Pivotal Software, Inc.

**Comment by avade** [comment]
Yes, this was happening every time. I added some additional debugging to the `cluster/scripts/install-from-tar` script and it was failing when untarring the cflinuxfs2.tar.gz. I then realized that the gitignored build dir is used by the `install-from-tar` script to store and cache cflinuxfs2.tar.gz. In my case there was a partially downloaded `cflinuxfs2.tar.gz` file in the downloads directory, so the script did not re-download. I deleted the partial download, re-ran `vagrant provision`, all good :) 

Perhaps some enhanced logging around this would be useful or an md5 check, rather than just seeing if the file exists. Will take a look at develop, may submit a PR for consideration.
